### ISSUE TITLE
Slashes in headings removed instead of converted into hyphens in anchors

### ISSUE THREAD

**Issue by quinnkeast** [sourcegraph/handbook#231]
Slashes `/` are removed rather than converted into `-` in anchors.

For example, on pages like the [Teammates](https://handbook.sourcegraph.com/company/team) page, the `she/her` is converted into `sheher` instead of the previous `she-her`, which breaks existing links to team members using this format.

**Comment by felixfbecker** [comment]
I think we should change the links instead of changing the ID generation, since the [new plugin we use](https://github.com/remarkjs/remark-slug) follows the exact algorithm GitHub uses for markdown heading IDs.

Aside: I wonder if it would be easier to have pronouns in the info below the heading instead of in the heading itself. Many times I've wanted to link to a teammate and had to look up the page and copy-paste the link because it's hard to remember how the IDs are generated in this case. If it was just names, it would be easy to write by hand in most cases.

**Comment by quinnkeast** [comment]
Makes sense to me! How about we just change the links to start with, and then look at the rest of it later?

**Comment by felixfbecker** [comment]
Yeah, would probably just be a quick search+replace with an IDE.

**Comment by marybelzer** [comment]
@felixfbecker @quinnkeast Now that the [team page](https://handbook.sourcegraph.com/team) is generated from a yaml file, is this still an issue?
